### ISSUE TITLE
Error when resolving dependencies in Windows

### ISSUE THREAD

**Issue by lukaseder** [nasven/nasven#6]
I'm getting this cryptic error when a nasven project declares dependencies

```
Lukas@LUKAS-ENVY ~/workspace/nasven
$ jjs -scripting nasven/nasven.js -- samples/jaxrs
[NASVEN] Building temporary Apache Maven project to find dependencies ...
Exception in thread "main" java.lang.RuntimeException: java.io.IOException: Cannot run program "mvn" (in directory "c:\Users\Lukas\workspace\nasven\samples\jaxr
s"): CreateProcess error=2, Das System kann die angegebene Datei nicht finden
        at jdk.nashorn.internal.runtime.ScriptRuntime.apply(ScriptRuntime.java:391)
        at jdk.nashorn.tools.Shell.apply(Shell.java:394)
        at jdk.nashorn.tools.Shell.runScripts(Shell.java:323)
        at jdk.nashorn.tools.Shell.run(Shell.java:169)
        at jdk.nashorn.tools.Shell.main(Shell.java:133)
        at jdk.nashorn.tools.Shell.main(Shell.java:112)
Caused by: java.io.IOException: Cannot run program "mvn" (in directory "c:\Users\Lukas\workspace\nasven\samples\jaxrs"): CreateProcess error=2, Das System kann
die angegebene Datei nicht finden
        at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
        at jdk.nashorn.internal.scripts.Script$Recompilation$9$6299AAZ$nasven.Nasven$__exec_input(nasven/nasven.js:164)
        at jdk.nashorn.internal.scripts.Script$Recompilation$8$6135AZ$nasven.Nasven$exec(nasven/nasven.js:140)
        at jdk.nashorn.internal.scripts.Script$Recompilation$5$2900A$nasven.Nasven$buildClasspath(nasven/nasven.js:108)
        at jdk.nashorn.internal.scripts.Script$Recompilation$1$1105$nasven.Nasven(nasven/nasven.js:128)
        at jdk.nashorn.internal.scripts.Script$nasven.:program(nasven/nasven.js:29)
        at jdk.nashorn.internal.runtime.ScriptFunctionData.invoke(ScriptFunctionData.java:636)
        at jdk.nashorn.internal.runtime.ScriptFunction.invoke(ScriptFunction.java:229)
        at jdk.nashorn.internal.runtime.ScriptRuntime.apply(ScriptRuntime.java:387)
        ... 5 more
Caused by: java.io.IOException: CreateProcess error=2, Das System kann die angegebene Datei nicht finden
        at java.lang.ProcessImpl.create(Native Method)
        at java.lang.ProcessImpl.<init>(ProcessImpl.java:386)
        at java.lang.ProcessImpl.start(ProcessImpl.java:137)
        at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
```

The error doesn't appear when I run a dependency-less project:

```
Lukas@LUKAS-ENVY ~/workspace/nasven
$ jjs -scripting nasven/nasven.js -- samples/helloworld
[NASVEN] About to run your nasven.js application under c:\Users\Lukas\workspace\nasven\samples\helloworld\index.js ...

[NASVEN] Calling jjs for your application ...
Hello Nashorn Maven Executor

undefined
[NASVEN] Application successfuly executed.
```

Is this perhaps a bad setup on my part? But even then, a nicer error message would be great :)

**Comment by lukaseder** [comment]
**_Notes**_

Maven is on the path:

```
Lukas@LUKAS-ENVY ~/workspace/nasven
$ echo $PATH
...:/c/apache-maven-3.2.3/bin:...
```

As can be seen here:

```
Lukas@LUKAS-ENVY ~/workspace/nasven
$ mvn
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.093 s
[INFO] Finished at: 2015-07-29T17:41:31+02:00
[INFO] Final Memory: 7M/232M
[INFO] ------------------------------------------------------------------------
```

Note that the command line syntax is weird because I'm using the git bash shell, but the same issue happens in the ordinary windows command line.

**Comment by lukaseder** [comment]
For the record, the missing piece is how to call `mvn` on Windows using Java's `ProcessBuilder`. This can be seen here: http://stackoverflow.com/a/17120829/521799

**Comment by brunoborges** [comment]
Fixed by PR #7
